.gitlab-ci.yml

```diff
@@ -3675,12 +3927,16 @@ latest_release_7:
     - "REPO=yum PATTERN_SUBSTRING=/$DEB_RPM_BUCKET_BRANCH/ ./invalidate.sh"
 
 deploy_cloudfront_invalidate_on_success:
+  rules:
+    - <<: *if_triggered_on_tag_7
+      when: on_success
```

Comment:
Why is this `if_triggered_on_tag_7` and not `if_trigger`? I expected it to be `if_trigger` since I thought you were replacing the `run_only_when_triggered` from `deploy_cloudfront_invalidate`.